README.md

```diff
@@ -12,12 +12,14 @@ plugins : {
     'verify-screenshot' : {
         testBasePath : '/func-test/tests',
         referencePath : '/func-test/screenshots',
-        diifPath : '/func-test/screenshots-diff'
+        newPath : '/func-test/screenshots-new',
```

Comment:
What about renaming it to `badPath` or `unmatchedPath`? I'm not sure `new` is an obvious word for this